lib/ansible/modules/storage/netapp/netapp_e_host.py

```diff
@@ -25,44 +24,141 @@
+            - If the host doesn't yet exist, the label/name to assign at creation time.
+            - If the hosts already exists, this will be used to uniquely identify the host to make any required changes
         required: True
+        aliases:
+            - label
+    state:
+        description:
+            - Set to absent to remove an existing host
+            - Set to present to modify or create a new host definition
+        choices:
+            - absent
+            - present
+        default: present
+        version_added: 2.6
     host_type_index:
         description:
             - The index that maps to host type you wish to create. It is recommended to use the M(netapp_e_facts) module to gather this information.
               Alternatively you can use the WSP portal to retrieve the information.
-        required: True
+            - Required when C(state=present)
+        aliases:
+            - host_type
     ports:
         description:
-            - a list of of dictionaries of host ports you wish to associate with the newly created host
+            - A list of host ports you wish to associate with the host.
+            - Host ports are uniquely identified by their WWN or IQN. Their assignments to a particular host are
+             uniquely identified by a label and these must be unique.
         required: False
+        suboptions:
+            type:
+                description:
+                  - The interface type of the port to define.
+                  - Acceptable choices depend on the capabilities of the target hardware/software platform.
+                required: true
+                choices:
+                  - iscsi
+                  - sas
+                  - fc
+                  - ib
+                  - nvmeof
+                  - ethernet
+            label:
+                description:
+                    - A unique label to assign to this port assignment.
+                required: true
+            port:
+                description:
+                    - The WWN or IQN of the hostPort to assign to this port definition.
+                required: true
+    force_port:
+        description:
+            - Allow ports that are already assigned to be re-assigned to your current host
+        required: false
+        type: bool
+        version_added: 2.6
     group:
         description:
-            - the group you want the host to be a member of
+            - The unique identifier of the host-group you want the host to be a member of; this is used for clustering.
         required: False
+        aliases:
+            - cluster
+    log_path:
+        description:
+            - A local path to a file to be used for debug logging
+        required: False
+        version_added: 2.6
 
 """
 
 EXAMPLES = """
-    - name: Set Host Info
+    - name: Define or update an existing host named 'Host1'
+      netapp_e_host:
+        ssid: "1"
```

Comment:
Yes. I thought that having real examples rather than variable references would be more useful. Although the IP address was real, everything else was intentionally faked.